features/events/traps/src/main/java/org/opennms/netmgt/trapd/TrapdConfigBean.java

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
```

Comment:
I was looking at SyslogD code & saw that there is a implementation class for SyslogConfig interface so I created it for TrapD as well.